src/stories/Listbox.mdx

```diff
@@ -230,3 +230,16 @@ via `panelClasses` prop. Remember to add `overflow-auto` along with your height
 <Canvas>
   <Story id="components-listbox--panel-classes" />
 </Canvas>
+
+### Inside a container with overflow-auto
```

Comment:
Not sure why, I can't find this story in the listbox block https://649d652b7201581ed9f63642-wjosvtoqui.chromatic.com/?path=/docs/components-listbox--panel-classes. (expect to see it after the components-listbox--panel-classes).
<img width="214" alt="image" src="https://github.com/archilogic-com/honeycomb/assets/7584821/8f64cecb-90f9-46cc-8240-bb25e5791553">